scripts/generate-sql.ts

```diff
@@ -76,7 +76,11 @@ function generateSQL(out?: string) {
 	const sqlPathsBefore = new Set(readdirSync(out).filter((file) => file.endsWith(".sql")));
 
 	// Run the command to generate the SQL.
-	sh(`yarn drizzle-kit generate --config ${configPath}`);
+	// The `generate' command sometimes prints errors, or becomes interactive and waits for user
```

Comment:
If the command waits for user input, then the process will hang, the `sh` call won't return anything and nothing will be logged.